### PR TITLE
Checked and corrected the nl-NL translations

### LINKED ISSUE #873: TASK: Check nl-NL translation
## Is your feature request related to a problem?
**Please describe.**
Check the nl-NL translation for issues

### PR DESCRIPTION
**Pull request by Timo-Breumelhof**
### Description of PR...
Corrected nl-NL language pack

## How did you test these updates?  

Local install

## PR Template Checklist

- [ ] Fixes Bug
- [ ] Feature solution
- [x] Other
- [ ] Requires documentation updates  
- [ ] I've updated the documentation already  

## Please mark which issue is solved

Close #873

### CODE REVIEW

File: Dnn.CommunityForums/App_LocalResources/ControlPanel.ascx.nl-NL.resx [hunk 1]
```diff
@@ -573,14 +678,14 @@
...
-  <data name="[RESX:MainSettings:Intro].Text" xml:space="preserve" lastModified="2024-05-16 19:43:03">
+  <data name="[RESX:MainSettings:Intro].Text" xml:space="preserve">
     <value> </value>
   </data>
   <data name="[RESX:MainSettings].Text" xml:space="preserve">
     <value>HoofdInstellingen</value>
   </data>
   <data name="[RESX:Maint:Days].Text" xml:space="preserve">
-    <value>dagen</value>
+    <value>dagen gelden</value>
```
Comment (Timo-Breumelhof) [inline Dnn.CommunityForums/App_LocalResources/ControlPanel.ascx.nl-NL.resx:688]: This should be "dagen geleden"

**Review by Timo-Breumelhof — COMMENTED** [review]
@johnhenley I found one issue, see other comment.
Rest LGTM

**Comment by Timo-Breumelhof** [comment]
@johnhenley oh sorry, I got an "review this " email, but it's already merged.. I think that one issue is also already fixed

**Comment by johnhenley** [comment]
> @johnhenley oh sorry, I got an "review this " email, but it's already merged.. I think that one issue is also already fixed

Sorry I thought it was still open

**Pull request merged**